pkg/action/scan.go

```diff
@@ -147,12 +155,35 @@ func errIfHitOrMiss(frs map[string]*bincapz.FileReport, kind string, scanPath st
 	return nil
 }
 
+// structs and types for sorting reports alphabetically by path
+// kv stores the keys and values of a map.
+type kv struct {
```

Comment:
This could use a better name. If I saw `actions.kv` somewhere, I would not expect it to hold file reports.

Question: what makes this path different than the one already included in FileReport? If they are the same, we should just sort by that instead.

Reply:
Removed in [`c03d232` (#405)](https://github.com/chainguard-dev/bincapz/pull/405/commits/c03d232eece70d75d3ed81466e19bfbdb8af0423).